molecule/default/tests/test_default.py

```diff
@@ -7,25 +7,18 @@
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
 
 
+@pytest.mark.skip(reason='TODO: will update the test to run '
+                         'cincer command on controller')
```

Comment:
You must `include pytest` prior to calling `@pytest.mark.skip`, or it will it will not be available resulting in the following error:
```
E   NameError: name 'pytest' is not defined
```

Also typo in above line: `s/cincer/cinder/`